### ISSUE TITLE
Can NOT use "npm install bleno" in WRTnode, which is a kind of openwrt system.

### ISSUE THREAD

**Issue by kaiser-ren** [sandeepmistry/bleno#108]
my platform is WRTnnode, a kind of openwrt syste. www.wrtnode.com
I had installed nodejs.
Then when I use npm install bleno, the installer reported an error:
npm ERR! bleno@0.1.13 install: `node-gyp rebuild`
npm ERR! Exit status 137
npm ERR!
npm ERR! Failed at the bleno@0.1.13 install script.
npm ERR! This is most likely a problem with the bleno package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls bleno
npm ERR! There is likely additional logging output above.
npm ERR! System Linux 3.10.49
npm ERR! command "node" "/usr/bin/npm" "install" "bleno"
npm ERR! cwd /tmp
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE

what's wrong with that?

**Comment by sandeepmistry** [comment]
Can you please share the npm debuf log? Also, do you have the libbluetooth headers installed?

**Comment by kaiser-ren** [comment]
Hi Weekly Status Report--20150615

**Comment by kaiser-ren** [comment]
I have tried to install "libbluetooth-dev", but it's unavailable in my platform, WRTnode. I can install "libbluetooth-dev" in Raspberry Pi.
As per npm debug log, how can I get it from my board?

**Comment by sandeepmistry** [comment]
You'll have to find the equivalent to install.

You could just use ```cat npm-debug.log``` and copy/paste the output from the console.

**Comment by kaiser-ren** [comment]
I can use "npm list" in /root to list the install packages, I found bleno was installed in my WRTnode.
-bleno@0.1.13
--bplist-parser@0.0.6
--debug@0.7.4

**Comment by kaiser-ren** [comment]
After that, I copy example/battery_service into my WRTnode, and run "npm install" in folder "battery_service", but below message pop-up.
npm WARN package.json battery-service@0.1.0 No repository field.
npm WARN optional dep failed, continuing xpc-connection@0.1.3

> bleno@0.1.13 install /kai/battery-service/node_modules/bleno
> node-gyp rebuild




#
# Fatal error in ../deps/v8/src/incremental-marking.cc, line 458
# CHECK(success) failed
#


==== Stack trace ============================================

Security context: 0x35f09619 <JS Object>#0#
    1: emit [events.js:53] (this=0x2b1f7d71 <a BlockStream>#1#,type=0x2b70bf6d <String[4]: data>)
    2: arguments adaptor frame: 2->1
    3: _emitChunk [/usr/lib/node_modules/npm/node_modules/block-stream/block-stream.js:145] (this=0x2b1f7d71 <a BlockStream>#1#,flush=0x35f08091 <undefined>)
    4: arguments adaptor frame: 0->1
    5: resume [/usr/lib/node_modules/npm/node_modules/block-stream/block-stream.js:58] (this=0x2b1f7d71 <a BlockStream>#1#)
    6: arguments adaptor frame: 1->0
    7: resume [/usr/lib/node_modules/npm/node_modules/fstream/lib/reader.js:255] (this=0x2b1f7dc1 <an Extract>#2#,who=0x2b1f7dc1 <an Extract>#2#)
    8: arguments adaptor frame: 0->1
    9: ondrain [stream.js:61] (this=0x2b1a6be1 <a DirWriter>#3#)
   10: emit [events.js:92] (this=0x2b1a6be1 <a DirWriter>#3#,type=0x3f4a5455 <String[5]: drain>)
   11: _process [/usr/lib/node_modules/npm/node_modules/fstream/lib/dir-writer.js:88] (this=0x2b1a6be1 <a DirWriter>#3#)
   12: onend [/usr/lib/node_modules/npm/node_modules/fstream/lib/dir-writer.js:169] (this=0x3470819d <a FileWriter>#4#)
   13: emit [events.js:92] (this=0x3470819d <a FileWriter>#4#,type=0x2b7193b9 <String[5]: close>)
   14: statCb [/usr/lib/node_modules/npm/node_modules/fstream/lib/writer.js:135] (this=0x35f096c1 <JS Global Object>#5#,er=0x34708245 <an Error>#6#,current=0x35f08091 <undefined>)
   15: arguments adaptor frame: 1->2
   17: oncomplete [evalmachine.<anonymous>:107] (this=0x347082a5 <an Object>#7#)
   18: arguments adaptor frame: 1->0

==== Details ================================================

[1]: emit [events.js:53] (this=0x2b1f7d71 <a BlockStream>#1#,type=0x2b70bf6d <String[4]: data>) {
  // stack-allocated locals
  var arguments = 0x35f08091 <undefined>
  var er = 0x35f08091 <undefined>
  var handler = 0x35f08091 <undefined>
  var len = 0x35f08091 <undefined>
  var args = 0x35f08091 <undefined>
  var i = 0x35f08091 <undefined>
  var listeners = 0x35f08091 <undefined>
  // heap-allocated locals
  var type = 0x2b70bf6d <String[4]: data>
  // expression stack (top to bottom)
  [10] : 2
  [09] : 1069774640
  [08] : 0x35f24431 <JS Function emit>#8#
  [07] : 0x35f08091 <undefined>
--------- s o u r c e   c o d e ---------
function emit(type) {?  var er, handler, len, args, i, listeners;??  if (!this._events)?    this._events = {};??  // If there is no 'error' event listener then throw.?  if (type === 'error') {?    if (!this._events.error ||?        (typeof this._events.error === 'object' &&?         !this._events.error.length)) ...

-----------------------------------------
}

[2]: arguments adaptor frame: 2->1 {
  // actual arguments
  [00] : 0x2b70bf6d <String[4]: data>
  [01] : 0x3470809d <a Buffer>#9#  // not passed to callee
}

[3]: _emitChunk [/usr/lib/node_modules/npm/node_modules/block-stream/block-stream.js:145] (this=0x2b1f7d71 <a BlockStream>#1#,flush=0x35f08091 <undefined>) {
  // stack-allocated locals
  var padBytes = 0x35f08091 <undefined>
  var bufferIndex = 4
  var out = 0x3470809d <a Buffer>#9#
  var outOffset = 0
  var outHas = 0
  var cur = 0x347080b5 <a Buffer>#10#
  var curHas = 10240
  var l = 0x35f08091 <undefined>
  var i = 0x35f08091 <undefined>
  // expression stack (top to bottom)
  [11] : 0x3470809d <a Buffer>#9#
  [10] : 0x2b70bf6d <String[4]: data>
  [09] : 0x2b1f7d71 <a BlockStream>#1#
--------- s o u r c e   c o d e ---------
function (flush) {?  // debug("emitChunk flush=%j emitting=%j paused=%j", flush, this._emitting, this._paused)??  // emit a <chunkSize> chunk?  if (flush && this._zeroes) {?    // debug("    BS push zeroes", this._bufferLength)?    // push a chunk of zeroes?    var padBytes = (this._bufferLength % this._chun...

-----------------------------------------
}

[4]: arguments adaptor frame: 0->1 {
}

[5]: resume [/usr/lib/node_modules/npm/node_modules/block-stream/block-stream.js:58] (this=0x2b1f7d71 <a BlockStream>#1#) {
  // expression stack (top to bottom)
  [00] : 0x2b1f7d71 <a BlockStream>#1#
--------- s o u r c e   c o d e ---------
function () {?  // debug("   BS resume")?  this._paused = false?  return this._emitChunk()?}
-----------------------------------------
}

[6]: arguments adaptor frame: 1->0 {
  // actual arguments
  [00] : 0x2b1f7dc1 <an Extract>#2#  // not passed to callee
}

[7]: resume [/usr/lib/node_modules/npm/node_modules/fstream/lib/reader.js:255] (this=0x2b1f7dc1 <an Extract>#2#,who=0x2b1f7dc1 <an Extract>#2#) {
  // expression stack (top to bottom)
  [01] : 0x2b1f7dc1 <an Extract>#2#
  [00] : 0x2b1f7d71 <a BlockStream>#1#
--------- s o u r c e   c o d e ---------
function (who) {?  this._paused = false?  who = who || this?  this.emit("resume", who)?  if (this._stream) this._stream.resume(who)?  this._read()?}
-----------------------------------------
}

[8]: arguments adaptor frame: 0->1 {
}

[9]: ondrain [stream.js:61] (this=0x2b1a6be1 <a DirWriter>#3#) {
  // expression stack (top to bottom)
  [00] : 0x2b1f7dc1 <an Extract>#2#
--------- s o u r c e   c o d e ---------
function ondrain() {?    if (source.readable && source.resume) {?      source.resume();?    }?  }
-----------------------------------------
}

[10]: emit [events.js:92] (this=0x2b1a6be1 <a DirWriter>#3#,type=0x3f4a5455 <String[5]: drain>) {
  // stack-allocated locals
  var arguments = 0x347080cd <an Arguments>#11#
  var er = 0x35f08091 <undefined>
  var handler = 0x47ab16a9 <JS Function ondrain>#12#
  var len = 0x35f08091 <undefined>
  var args = 0x35f08091 <undefined>
  var i = 0x35f08091 <undefined>
  var listeners = 0x35f08091 <undefined>
  // heap-allocated locals
  var type = 0x3f4a5455 <String[5]: drain>
  // expression stack (top to bottom)
  [08] : 0x2b1a6be1 <a DirWriter>#3#
  [07] : 0x35f24431 <JS Function emit>#8#
--------- s o u r c e   c o d e ---------
function emit(type) {?  var er, handler, len, args, i, listeners;??  if (!this._events)?    this._events = {};??  // If there is no 'error' event listener then throw.?  if (type === 'error') {?    if (!this._events.error ||?        (typeof this._events.error === 'object' &&?         !this._events.error.length)) ...

-----------------------------------------
}

[11]: _process [/usr/lib/node_modules/npm/node_modules/fstream/lib/dir-writer.js:88] (this=0x2b1a6be1 <a DirWriter>#3#) {
  // stack-allocated locals
  var p = 0x35f08091 <undefined>
  var pp = 0x35f08091 <undefined>
  var onend = 0x347080fd <JS Function onend>#13#
  // heap-allocated locals
  var me = 0x2b1a6be1 <a DirWriter>#3#
  var entry = 0x35f08091 <undefined>
  var props = 0x35f08091 <undefined>
  var child = 0x35f08091 <undefined>
  var ended = 0x35f08091 <undefined>
  // expression stack (top to bottom)
  [04] : 0x3f4a5455 <String[5]: drain>
  [03] : 0x2b1a6be1 <a DirWriter>#3#
--------- s o u r c e   c o d e ---------
function () {?  var me = this??  // console.error("DW Process p=%j", me._processing, me.basename)??  if (me._processing) return??  var entry = me._buffer.shift()?  if (!entry) {?    // console.error("DW Drain")?    me.emit("drain")?    if (me._ended) me._finish()?    return?  }??  me._processing = true?  // ...

-----------------------------------------
}

[12]: onend [/usr/lib/node_modules/npm/node_modules/fstream/lib/dir-writer.js:169] (this=0x3470819d <a FileWriter>#4#) {
  // expression stack (top to bottom)
  [00] : 0x2b1a6be1 <a DirWriter>#3#
--------- s o u r c e   c o d e ---------
function onend() {?    if (ended) return?    ended = true?    // console.error("* DW Child end", child.basename)?    me._currentChild = null?    me._processing = false?    me._process()?  }
-----------------------------------------
}

[13]: emit [events.js:92] (this=0x3470819d <a FileWriter>#4#,type=0x2b7193b9 <String[5]: close>) {
  // stack-allocated locals
  var arguments = 0x347081d1 <an Arguments>#14#
  var er = 0x35f08091 <undefined>
  var handler = 0x3470814d <JS Function onend>#15#
  var len = 0x35f08091 <undefined>
  var args = 0x35f08091 <undefined>
  var i = 0x35f08091 <undefined>
  var listeners = 0x35f08091 <undefined>
  // heap-allocated locals
  var type = 0x2b7193b9 <String[5]: close>
  // expression stack (top to bottom)
  [08] : 0x3470819d <a FileWriter>#4#
  [07] : 0x35f24431 <JS Function emit>#8#
--------- s o u r c e   c o d e ---------
function emit(type) {?  var er, handler, len, args, i, listeners;??  if (!this._events)?    this._events = {};??  // If there is no 'error' event listener then throw.?  if (type === 'error') {?    if (!this._events.error ||?        (typeof this._events.error === 'object' &&?         !this._events.error.length)) ...

-----------------------------------------
}

[14]: statCb [/usr/lib/node_modules/npm/node_modules/fstream/lib/writer.js:135] (this=0x35f096c1 <JS Global Object>#5#,er=0x34708245 <an Error>#6#,current=0x35f08091 <undefined>) {
  // stack-allocated locals
  var currentType = 0x35f08091 <undefined>
  // expression stack (top to bottom)
  [02] : 0x2b7193b9 <String[5]: close>
  [01] : 0x3470819d <a FileWriter>#4#
--------- s o u r c e   c o d e ---------
function statCb(er, current) {?    if (me.filter && !me.filter.call(who, who, current)) {?      me._aborted = true?      me.emit("end")?      me.emit("close")?      return?    }??    // if it's not there, great.  We'll just create it.?    // if it is there, then we'll need to change whatever differs?    if (er || ...

-----------------------------------------
}

[15]: arguments adaptor frame: 1->2 {
  // actual arguments
  [00] : 0x34708245 <an Error>#6#
}

[17]: oncomplete [evalmachine.<anonymous>:107] (this=0x347082a5 <an Object>#7#) {
  // stack-allocated locals
  var arguments = 0x34708251 <an Arguments>#16#
  // expression stack (top to bottom)
  [03] : 0x34708251 <an Arguments>#16#
  [02] : 0x35f08081 <null>
  [01] : 0x34708201 <JS Function statCb>#17#
--------- s o u r c e   c o d e ---------
function () {?    return cb.apply(null, arguments);?  }
-----------------------------------------
}

[18]: arguments adaptor frame: 1->0 {
  // actual arguments
  [00] : 0x34708245 <an Error>#6#  // not passed to callee
}

==== Key         ============================================

#0# 0x35f09619: 0x35f09619 <JS Object>
#1# 0x2b1f7d71: 0x2b1f7d71 <a BlockStream>
          readable: 0x35f080a1 <true>
          writable: 0x35f080a1 <true>
              _opt: 0x2b1a8165 <an Object>#18#
        _chunkSize: 512
           _offset: 6656
           _buffer: 0x34708385 <JS Array[9]>#19#
     _bufferLength: 58880
           _zeroes: 0x2b1a8171 <a Buffer>#20#
           _events: 0x2b1a8189 <an Object>#21#
           _paused: 0x35f080b1 <false>
         _emitting: 0x35f080a1 <true>
       _needsDrain: 0x35f080a1 <true>
        _needDrain: 0x35f080a1 <true>
#2# 0x2b1f7dc1: 0x2b1f7dc1 <an Extract>
            domain: 0x35f08081 <null>
           _events: 0x2b1a81c5 <an Object>#22#
     _maxListeners: 10
          writable: 0x35f080a1 <true>
          readable: 0x35f080a1 <true>
           _stream: 0x2b1f7d71 <a BlockStream>#1#
          position: 11458560
            _ended: 0x35f080b1 <false>
              _fst: 0x2b1a6be1 <a DirWriter>#3#
           _paused: 0x35f080b1 <false>
       _eofStarted: 0x35f080b1 <false>
         _extended: 0x35f08081 <null>
            _entry: 0x34708609 <an Entry>#23#
#3# 0x2b1a6be1: 0x2b1a6be1 <a DirWriter>
            domain: 0x35f08081 <null>
           _events: 0x47ab12f5 <an Object>#24#
     _maxListeners: 10
              type: 0x5339c061 <String[9]: Directory>
             props: 0x47ab1301 <an Object>#25#
             depth: 0
           clobber: 0x35f080a1 <true>
            parent: 0x35f08081 <null>
              root: 0x2b1a6be1 <a DirWriter>#3#
              path: 0x4f32edc1 <String[23]: /root/.node-gyp/0.10.33>
             _path: 0x4f32edc1 <String[23]: /root/.node-gyp/0.10.33>
          basename: 0x4f32f0e5 <String[7]: 0.10.33>
           dirname: 0x2b142a5d <String[15]: /root/.node-gyp>
          linkpath: 0x35f08081 <null>
              size: 0x35f08091 <undefined>
          readable: 0x35f080b1 <false>
          writable: 0x35f080a1 <true>
           _buffer: 0x2b142a71 <JS Array[0]>#26#
             ready: 0x35f080a1 <true>
              _old: 0x2b142a81 <JS Object>#27#
          _madeDir: 0x35f08081 <null>
       _processing: 0x35f080b1 <false>
     _currentChild: 0x35f08081 <null>
#4# 0x3470819d: 0x3470819d <a FileWriter>
           _buffer: 0x34722495 <JS Array[0]>#28#
     _bytesWritten: 0
            domain: 0x35f08081 <null>
           _events: 0x347224a5 <an Object>#29#
     _maxListeners: 10
              type: 0x5339caa9 <String[4]: File>
             props: 0x34722441 <an Object>#30#
             depth: 1
           clobber: 0x35f080a1 <true>
            parent: 0x2b1a6be1 <a DirWriter>#3#
              root: 0x2b1a6be1 <a DirWriter>#3#
              path: 0x34724a8d <String[57]: /root/.node-gyp/0.10.33/deps/openssl/openssl/ssl/t1_lib.c>
             _path: 0x34724a8d <String[57]: /root/.node-gyp/0.10.33/deps/openssl/openssl/ssl/t1_lib.c>
          basename: 0x34724b0d <String[8]: t1_lib.c>
           dirname: 0x34724b29 <String[48]: /root/.node-gyp/0.10.33/deps/openssl/openssl/ssl>
          linkpath: 0x35f08081 <null>
              size: 75770
          readable: 0x35f080b1 <false>
          writable: 0x35f080a1 <true>
             ready: 0x35f080b1 <false>
          _aborted: 0x35f080a1 <true>
#5# 0x35f096c1: 0x35f096c1 <JS Global Object>
#6# 0x34708245: 0x34708245 <an Error>
         arguments: 0x35f08091 <undefined>
              type: 0x35f08091 <undefined>
           message: 0x34708349 <String[73]: ENOENT, lstat '/root/.node-gyp/0.10.33/deps/openssl/openssl/ssl/t1_lib.c'>
             errno: 34
              code: 0x2b7233ed <String[6]: ENOENT>
              path: 0x347082d9 <String[57]: /root/.node-gyp/0.10.33/deps/openssl/openssl/ssl/t1_lib.c>
#7# 0x347082a5: 0x347082a5 <an Object>
            domain: 0x35f08081 <null>
        oncomplete: 0x34708265 <JS Function>#31#
#8# 0x35f24431: 0x35f24431 <JS Function emit>
#9# 0x3470809d: 0x3470809d <a Buffer>
            length: 512
            parent: 0x34722411 <a SlowBuffer>#32#
            offset: 6144
#10# 0x347080b5: 0x347080b5 <a Buffer>
            length: 16384
            parent: 0x34722411 <a SlowBuffer>#32#
            offset: 0
#11# 0x347080cd: 0x347080cd <an Arguments>
            length: 1
            callee: 0x35f24431 <JS Function emit>#8#
#12# 0x47ab16a9: 0x47ab16a9 <JS Function ondrain>
#13# 0x347080fd: 0x347080fd <JS Function onend>
#14# 0x347081d1: 0x347081d1 <an Arguments>
            length: 1
            callee: 0x35f24431 <JS Function emit>#8#
#15# 0x3470814d: 0x3470814d <JS Function onend>
#16# 0x34708251: 0x34708251 <an Arguments>
            length: 1
            callee: 0x34708265 <JS Function>#31#
#17# 0x34708201: 0x34708201 <JS Function statCb>
#18# 0x2b1a8165: 0x2b1a8165 <an Object>
#19# 0x34708385: 0x34708385 <JS Array[9]>
                 0: 0x34724b5d <a Buffer>#33#
                 1: 0x34724b75 <a Buffer>#34#
                 2: 0x34724b8d <a Buffer>#35#
                 3: 0x34724ba5 <a Buffer>#36#
                 4: 0x347080b5 <a Buffer>#10#
                 5: 0x34724bbd <a Buffer>#37#
                 6: 0x34724bd5 <a Buffer>#38#
                 7: 0x34724bed <a Buffer>#39#
                 8: 0x34724c05 <a Buffer>#40#
#20# 0x2b1a8171: 0x2b1a8171 <a Buffer>
            length: 512
            parent: 0x2b144c81 <a SlowBuffer>#41#
            offset: 7472
#21# 0x2b1a8189: 0x2b1a8189 <an Object>
#22# 0x2b1a81c5: 0x2b1a81c5 <an Object>
             entry: 0x2b142a1d <JS Array[2]>#42#
             error: 0x2b142a2d <JS Array[3]>#43#
               end: 0x2b142a3d <JS Array[2]>#44#
             close: 0x2b142a4d <JS Array[2]>#45#
#23# 0x34708609: 0x34708609 <an Entry>
            domain: 0x35f08081 <null>
           _events: 0x347225b5 <an Object>#46#
     _maxListeners: 10
          readable: 0x35f080a1 <true>
          writable: 0x35f080a1 <true>
        _needDrain: 0x35f080a1 <true>
           _paused: 0x35f080a1 <true>
          _reading: 0x35f080b1 <false>
           _ending: 0x35f080b1 <false>
            _ended: 0x35f080b1 <false>
        _remaining: 17402
            _queue: 0x347225c1 <JS Array[114]>#47#
            _index: 0
         _queueLen: 114
             _read: 0x347225d1 <JS Function>#48#
             props: 0x347225f5 <an Object>#49#
           _header: 0x34722601 <a TarHeader>#50#
         _extended: 0x34722635 <an Object>#51#
           _global: 0x34722641 <an Object>#52#
              type: 0x5339caa9 <String[4]: File>
              path: 0x3472264d <String[33]: deps/openssl/openssl/ssl/t1_lib.c>
              size: 75770
              meta: 0x35f080b1 <false>
        _collected: 0x35f080a1 <true>
              pipe: 0x34724c3d <JS Function>#53#
#24# 0x47ab12f5: 0x47ab12f5 <an Object>
            close: 0x50a6ef29 <JS Array[2]>#54#
#25# 0x47ab1301: 0x47ab1301 <an Object>
              path: 0x4f32efcd <String[23]: /root/.node-gyp/0.10.33>
             strip: 1
            filter: 0x2b1a811d <JS Function isValid>#55#
              type: 0x5339c061 <String[9]: Directory>
         Directory: 0x35f080a1 <true>
              root: 0x35f08081 <null>
            parent: 0x35f08081 <null>
#26# 0x2b142a71: 0x2b142a71 <JS Array[0]>
#27# 0x2b142a81: 0x2b142a81 <JS Object>
               dev: 2049
              mode: 16877
             nlink: 2
               uid: 0
               gid: 0
              rdev: 0
           blksize: 4096
               ino: 3277
              size: 4096
            blocks: 8
             atime: 0x47a5debd <a Date>#56#
             mtime: 0x47a5deed <a Date>#57#
             ctime: 0x47a5df1d <a Date>#58#
         Directory: 0x35f080a1 <true>
              type: 0x5339c061 <String[9]: Directory>
#28# 0x34722495: 0x34722495 <JS Array[0]>
#29# 0x347224a5: 0x347224a5 <an Object>
             ready: 0x34728641 <JS Function>#59#
             error: 0x34728665 <JS Function>#60#
             close: 0x3470814d <JS Function onend>#15#
#30# 0x34722441: 0x34722441 <an Object>
            parent: 0x35f08081 <null>
              root: 0x35f08081 <null>
              type: 0x5339caa9 <String[4]: File>
             depth: 1
              path: 0x3472857d <String[57]: /root/.node-gyp/0.10.33/deps/openssl/openssl/ssl/t1_lib.c>
              mode: 436
               uid: 0
               gid: 0
              size: 75770
             mtime: 0x347285fd <a Date>#61#
             cksum: 8149
          linkpath: 0x2b708185 <String[0]: >
             ustar: 0x35f080b1 <false>
              File: 0x35f080a1 <true>
#31# 0x34708265: 0x34708265 <JS Function>
#32# 0x34722411: 0x34722411 <a SlowBuffer>
            length: 16384
#33# 0x34724b5d: 0x34724b5d <a Buffer>
            length: 16384
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 0
#34# 0x34724b75: 0x34724b75 <a Buffer>
            length: 16384
            parent: 0x347286b1 <a SlowBuffer>#63#
            offset: 0
#35# 0x34724b8d: 0x34724b8d <a Buffer>
            length: 16384
            parent: 0x347286cd <a SlowBuffer>#64#
            offset: 0
#36# 0x34724ba5: 0x34724ba5 <a Buffer>
            length: 16384
            parent: 0x347286e9 <a SlowBuffer>#65#
            offset: 0
#37# 0x34724bbd: 0x34724bbd <a Buffer>
            length: 16384
            parent: 0x34728705 <a SlowBuffer>#66#
            offset: 0
#38# 0x34724bd5: 0x34724bd5 <a Buffer>
            length: 16384
            parent: 0x34728721 <a SlowBuffer>#67#
            offset: 0
#39# 0x34724bed: 0x34724bed <a Buffer>
            length: 6938
            parent: 0x3472873d <a SlowBuffer>#68#
            offset: 0
#40# 0x34724c05: 0x34724c05 <a Buffer>
            length: 9446
            parent: 0x3472873d <a SlowBuffer>#68#
            offset: 6938
#41# 0x2b144c81: 0x2b144c81 <a SlowBuffer>
            length: 8192
              used: 7984
#42# 0x2b142a1d: 0x2b142a1d <JS Array[2]>
                 0: 0x47ab1619 <JS Function>#69#
                 1: 0x47a5de75 <JS Function>#70#
#43# 0x2b142a2d: 0x2b142a2d <JS Array[3]>
                 0: 0x50a6e935 <JS Function onerror>#71#
                 1: 0x47ab3045 <JS Function cb>#72#
                 2: 0x47ab16cd <JS Function onerror>#73#
#44# 0x2b142a3d: 0x2b142a3d <JS Array[2]>
                 0: 0x47ab163d <JS Function afterTarball>#74#
                 1: 0x2b144835 <JS Function cleanup>#75#
#45# 0x2b142a4d: 0x2b142a4d <JS Array[2]>
                 0: 0x2b144835 <JS Function cleanup>#75#
                 1: 0x47a5de99 <JS Function g>#76#
#46# 0x347225b5: 0x347225b5 <an Object>
              data: 0x34728775 <JS Array[2]>#77#
             pause: 0x34728785 <JS Function>#78#
            resume: 0x347287a9 <JS Function>#79#
               end: 0x347287cd <JS Function save>#80#
             entry: 0x347287f1 <JS Function saveEntry>#81#
             proxy: 0x34728815 <JS Function proxyPause>#82#
#47# 0x347225c1: 0x347225c1 <JS Array[114]>
                 0: 0x34728839 <a Buffer>#83#
                 1: 0x34728851 <a Buffer>#84#
                 2: 0x34728869 <a Buffer>#85#
                 3: 0x34728881 <a Buffer>#86#
                 4: 0x34728899 <a Buffer>#87#
                 5: 0x347288b1 <a Buffer>#88#
                 6: 0x347288c9 <a Buffer>#89#
                 7: 0x347288e1 <a Buffer>#90#
                 8: 0x347288f9 <a Buffer>#91#
                 9: 0x34728911 <a Buffer>#92#
                  ...
#48# 0x347225d1: 0x347225d1 <JS Function>
            length: 0
#49# 0x347225f5: 0x347225f5 <an Object>
              path: 0x3472264d <String[33]: deps/openssl/openssl/ssl/t1_lib.c>
              mode: 436
               uid: 0
               gid: 0
              size: 75770
             mtime: 0x347285fd <a Date>#61#
             cksum: 8149
              type: 0x2b708855 <String[1]: 0>
          linkpath: 0x2b708185 <String[0]: >
             ustar: 0x35f080b1 <false>
#50# 0x34722601: 0x34722601 <a TarHeader>
             block: 0x34724f45 <a Buffer>#93#
        cksumValid: 0x35f080a1 <true>
              path: 0x34724f5d <String[47]: node-v0.10.33/deps/openssl/openssl/ssl/t1_lib.c>
              mode: 436
               uid: 0
               gid: 0
              size: 75770
             mtime: 0x34724f99 <Number: 1414086312>
             cksum: 8149
              type: 0x2b708855 <String[1]: 0>
          linkpath: 0x2b708185 <String[0]: >
             ustar: 0x35f080b1 <false>
#51# 0x34722635: 0x34722635 <an Object>
#52# 0x34722641: 0x34722641 <an Object>
#53# 0x34724c3d: 0x34724c3d <JS Function>
#54# 0x50a6ef29: 0x50a6ef29 <JS Array[2]>
                 0: 0x47ab1685 <JS Function>#94#
                 1: 0x2b144835 <JS Function cleanup>#75#
#55# 0x2b1a811d: 0x2b1a811d <JS Function isValid>
#56# 0x47a5debd: 0x47a5debd <a Date>
#57# 0x47a5deed: 0x47a5deed <a Date>
#58# 0x47a5df1d: 0x47a5df1d <a Date>
#59# 0x34728641: 0x34728641 <JS Function>
#60# 0x34728665: 0x34728665 <JS Function>
#61# 0x347285fd: 0x347285fd <a Date>
#62# 0x34728695: 0x34728695 <a SlowBuffer>
            length: 16384
#63# 0x347286b1: 0x347286b1 <a SlowBuffer>
            length: 16384
#64# 0x347286cd: 0x347286cd <a SlowBuffer>
            length: 16384
#65# 0x347286e9: 0x347286e9 <a SlowBuffer>
            length: 16384
#66# 0x34728705: 0x34728705 <a SlowBuffer>
            length: 16384
#67# 0x34728721: 0x34728721 <a SlowBuffer>
            length: 16384
#68# 0x3472873d: 0x3472873d <a SlowBuffer>
            length: 16384
#69# 0x47ab1619: 0x47ab1619 <JS Function>
#70# 0x47a5de75: 0x47a5de75 <JS Function>
#71# 0x50a6e935: 0x50a6e935 <JS Function onerror>
#72# 0x47ab3045: 0x47ab3045 <JS Function cb>
#73# 0x47ab16cd: 0x47ab16cd <JS Function onerror>
#74# 0x47ab163d: 0x47ab163d <JS Function afterTarball>
#75# 0x2b144835: 0x2b144835 <JS Function cleanup>
#76# 0x47a5de99: 0x47a5de99 <JS Function g>
          listener: 0x50a6e959 <JS Function onclose>#95#
#77# 0x34728775: 0x34728775 <JS Array[2]>
                 0: 0x3473457d <JS Function>#96#
                 1: 0x347287cd <JS Function save>#80#
#78# 0x34728785: 0x34728785 <JS Function>
#79# 0x347287a9: 0x347287a9 <JS Function>
#80# 0x347287cd: 0x347287cd <JS Function save>
#81# 0x347287f1: 0x347287f1 <JS Function saveEntry>
#82# 0x34728815: 0x34728815 <JS Function proxyPause>
#83# 0x34728839: 0x34728839 <a Buffer>
            length: 512
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 13312
#84# 0x34728851: 0x34728851 <a Buffer>
            length: 512
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 13824
#85# 0x34728869: 0x34728869 <a Buffer>
            length: 512
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 14336
#86# 0x34728881: 0x34728881 <a Buffer>
            length: 512
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 14848
#87# 0x34728899: 0x34728899 <a Buffer>
            length: 512
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 15360
#88# 0x347288b1: 0x347288b1 <a Buffer>
            length: 512
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 15872
#89# 0x347288c9: 0x347288c9 <a Buffer>
            length: 512
            parent: 0x347286b1 <a SlowBuffer>#63#
            offset: 0
#90# 0x347288e1: 0x347288e1 <a Buffer>
            length: 512
            parent: 0x347286b1 <a SlowBuffer>#63#
            offset: 512
#91# 0x347288f9: 0x347288f9 <a Buffer>
            length: 512
            parent: 0x347286b1 <a SlowBuffer>#63#
            offset: 1024
#92# 0x34728911: 0x34728911 <a Buffer>
            length: 512
            parent: 0x347286b1 <a SlowBuffer>#63#
            offset: 1536
#93# 0x34724f45: 0x34724f45 <a Buffer>
            length: 512
            parent: 0x34728695 <a SlowBuffer>#62#
            offset: 12800
#94# 0x47ab1685: 0x47ab1685 <JS Function>
#95# 0x50a6e959: 0x50a6e959 <JS Function onclose>
#96# 0x3473457d: 0x3473457d <JS Function>
=====================

Trace/breakpoint trap

npm ERR! bleno@0.1.13 install: `node-gyp rebuild`
npm ERR! Exit status 133
npm ERR!
npm ERR! Failed at the bleno@0.1.13 install script.
npm ERR! This is most likely a problem with the bleno package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls bleno
npm ERR! There is likely additional logging output above.
npm ERR! System Linux 3.10.49
npm ERR! command "node" "/usr/bin/npm" "install"
npm ERR! cwd /kai/battery-service
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! not ok code 0

**Comment by sandeepmistry** [comment]
@kaiser-ren this is starting to become out of scope of bleno. Your either ```fstream`` module`,```node-gyp``` or ```npm``` is very broken on your system. I suggest you raise an issue with one of those ...

**Comment by soonsebii** [comment]
@kaiser-ren You can use Noble on a WRTnode. 
https://github.com/soonsebii/custom-package/tree/master/lang/noble
